homeassistant/components/esphome/update.py

```diff
@@ -78,7 +78,7 @@ def __init__(
         super().__init__(coordinator=coordinator)
         assert entry_data.device_info is not None
         self._device_info = entry_data.device_info
-        self._attr_unique_id = f"{entry_data.entry_id}_update"
+        self._attr_unique_id = entry_data.entry_id
```

Comment:
Any reason to not use the `mac_address` here in case they delete and recreate?

Reply:
🤷 Sounds like a better idea I agree

Reply:
adjusted